### ISSUE TITLE
Add v1.0 documentation to readthedocs

### ISSUE THREAD

**Issue by timtroendle** [calliope-project/euro-calliope#155]
Can we add documentation for v1.0 retroactively to Read the docs?

Right now, when clicking on `stable` there is nothing.

**Comment by timtroendle** [comment]
@brynpickering any idea if and how this can be done?

**Comment by brynpickering** [comment]
I don't believe it can be, since v1.0 has no documentation infrastructure in the release. One option would be to incorporate docs and move the tag (possibly dangerous now that it's been around a while). I suspect stable will only be useful from `v1.1`. In the meantime I have deactivated `stable` in RTD.

**Comment by timtroendle** [comment]
After experimenting a little with read the docs, I think there may be a way. We could create a v1.0 release branch branching off of the v1.0 release (which I think we should absolutely not touch). We can then activate this branch in the read the docs and it should build the documentation nicely. We can try after releasing 1.1 whether that makes sense.
